services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -15,64 +15,63 @@ import (
 	"time"
 
 	"github.com/google/uuid"
-	"github.com/red-hat-storage/ocs-operator/v4/services/types"
+	"github.com/red-hat-storage/ocs-operator/v4/services"
 	"k8s.io/klog/v2"
 )
 
-const onboardingPrivateKeyFilePath = "/etc/private-key/key"
-
-func OnboardingTokensHandler(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
+const (
+	onboardingPrivateKeyFilePath = "/etc/private-key/key"
+	ContentTypeTextPlain         = "text/plain"
+)
 
-	var err error
+func HandleMessage(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
```

Comment:
`HandleRequest` might sound and suit better for what we are doing 🤔

Reply:
For me, Request is the vehicle, Message is the content we want to handle.